### DISCUSSION TITLE
nvim启动报错

### DISCUSSION THREAD

**Discussion by Dennis-Zhang-SH** [ayamir/nvimdots#937] [Q&A] [answered]
Failed to source `/home/dennis/.local/share/nvim/site/lazy/nvim-treesitter-textobjects/plugin/nvim-treesitter-textobjects.vim`

vim/_editor.lua:0: /home/dennis/.config/nvim/init.lua..nvim_exec2() called at /home/dennis/.config/nvim/init.lua:0../home/dennis/.local/share/nvim/site/lazy/nvim-treesitter-textobjects/plu
gin/nvim-treesitter-textobjects.vim, line 3: Vim(lua):E5108: Error executing lua /usr/share/nvim/runtime/lua/vim/treesitter/query.lua:609: Overriding trim!

**Comment by ayamir** [comment]
`TSUpdate`

**Reply by Dennis-Zhang-SH** [reply]
在nvim内部:TSUpdate吗？找不到这条命令

**Reply by ayamir** [reply]
![image](https://github.com/ayamir/nvimdots/assets/61657399/93bea859-d5d6-4891-82e9-6e6ff1f5d5b1)

**Reply by Dennis-Zhang-SH** [reply]
![image](https://github.com/ayamir/nvimdots/assets/51769674/9b02e2a9-8ade-4d24-ab14-23e1cfac79e8)

**Reply by Jint-lzxy** [reply]
You need to wait for a while before typing `:TSUpdate`, as loading treesitter takes time.

**Reply by charliie-dev** [reply, marked as answer]
or just remove the plugin folder and re-download everything

**Reply by Dennis-Zhang-SH** [reply]
the exact path is?

**Reply by charliie-dev** [reply]
https://github.com/ayamir/nvimdots/wiki/Issues#remove-nvimdots
